ts/project.ts

```diff
@@ -67,39 +68,36 @@ export function isProject(p: unknown): p is Project {
 
 type Nestable = Environment | Package | Executable | Check;
 
-type ProjectSettings = {
-  description: string;
-  defaultEnvironment?: Environment;
-  defaultExecutable?: Executable;
-};
```

Comment:
We either need to inline this or export it through `mod.ts` in order to get visibility of the options here in docs.deno.land.

I opted for inlining for now since it wasn't used anywhere else.